packages/components/addon/components/hds/side-nav/section/link.hbs

```diff
@@ -0,0 +1,31 @@
+<li class="hds-side-nav__list-item">
+  <Hds::Interactive
+    @href={{@href}}
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @icon={{@icon}}
+    @isRouteExternal={{@isRouteExternal}}
+    @current-when={{@current-when}}
+    class="hds-side-nav__item hds-side-nav__item--link"
+    ...attributes
+  >
+    {{#if @icon}}
+      <FlightIcon class="hds-side-nav__list-icon" @name={{@icon}} @stretched={{true}} />
+    {{/if}}
+
+    <span class="hds-side-nav-label hds-typography-body-200 hds-font-weight-medium">
+      {{@label}}
+    </span>
+
+    {{#if @count}}
```

Comment:
I was trying to follow what I had done in [Tabs](https://helios.hashicorp.design/components/tabs?tab=code).

Reply:
I see. But probably in this case, since there is also `@badge`, I would make it more obvious on what is what.
Let's discuss anyway later, in the naming session.

Reply:
Going to keep this as @count for now per my understanding from our discussions. We can revisit the question before merging if need be.